README.md

```diff
@@ -1,6 +1,8 @@
 Npgsql Entity Framework Core provider for PostgreSQL
 =============
 
+[![Stable NuGet](https://img.shields.io/nuget/v/Npgsql.EntityFrameworkCore.PostgreSQL.svg?label=stable%20nuget)](https://www.nuget.org/packages/Npgsql.EntityFrameworkCore.PostgreSQL) [![Unstable MyGet](https://img.shields.io/myget/npgsql-unstable/v/Npgsql.EntityFrameworkCore.PostgreSQL.svg?label=unstable%20nuget)](https://www.myget.org/feed/npgsql-unstable/package/nuget/Npgsql.EntityFrameworkCore.PostgreSQL) [![Join the chat at https://gitter.im/npgsql/npsgl](https://img.shields.io/badge/GITTER-JOIN%20CHAT-brightgreen.svg)](https://gitter.im/npgsql/npgsql)
```

Comment:
Maybe change `unstable nuget` to `unstable myget` in the link or replace `MyGet` by `NuGet`? I think the first would be better.